vmdb/app/models/vm_or_template/operations/smart_state.rb

```diff
@@ -0,0 +1,9 @@
+class VmOrTemplate
+  module Operations
+    module SmartState
+      def validate_scan
+        validate_vm_control_powered_on
```

Comment:
Hmm, was basing logic on app/models/vm_or_template.rb self.scan_by_property method (line ~1833) checks for on state before the scan.  Is that old code then and no longer true ?  If so I'll update this PR.